examples/wearable/UIComponents/contents/user-inputs-elements/controls/radio.html

```diff
@@ -14,38 +14,28 @@
 </head>
 
 <body>
-    <div class="ui-page">
+    <div class="ui-page" id="radio-demo">
         <header class="ui-header">
             <h2 class="ui-title">Radio</h2>
         </header>
         <div class="ui-content">
             <ul class="ui-listview">
                 <li class="li-has-radio">
-                    <label>
+                    <label id="radio-text-1">
                         Radio on
-                        <input type="radio" checked="checked" name="radio-el"/>
+                        <input type="radio" id="radio-1" checked="checked" name="radio-el"/>
                     </label>
                 </li>
                 <li class="li-has-radio">
-                    <label>
+                    <label id="radio-text-2"> 
                         Radio off
-                        <input type="radio" name="radio-el" />
-                    </label>
-                </li>
-                <li class="li-has-radio">
```

Comment:
This example shows how the disabled radio button looks. Do you sure that you want delete this example?

Reply:
It's SAD issue and I can't find the disabled radio button in the Guide.
And tester said it's a little confusing like duplicated button.
So I delete this example.

Reply:
Thank you for explanation... but disabled radio button is shown on page 87

Reply:
Oh.
I see the "HTML" file..
So how about the  like "Radio on disable" ," Radio off disable" to avoid confusion?
![W-5 0-circle-x86-2020-04-23-185845](https://user-images.githubusercontent.com/31196177/80086533-d0071300-8594-11ea-8de7-1f975b869cc6.png)

Reply:
I forgot that we are currently using the guide in the html version. There is actually no example exactly like our application

Reply:
How about the "disable" menu?
Is it better to remain the disable sample?

Reply:
I think that "Radio on disabled" ," Radio off disabled" will be ok but in the same menu as you proposed on screen

Reply:
Ok.
I add the "Radio on disabled" ," Radio off disabled".

Reply:
Radio buttons works good.